css/theme-topic.css

```diff
@@ -38,87 +38,99 @@
   font-size: 1.1em;
   margin-top: 10px;
 }
 
-.media-spc {
-  margin-left: 2em;
-  margin-top: 1em;
+.experimental .followus {
+  line-height: 0em;
+  background-color: #f5f5f5;
+
 }
-.media-spc-2 {
-  margin-left: 2em;
-  margin-top: 1em;
+.experimental .followus a {
+  text-decoration: none;
 }
 
-.facebook-2 {
-  background: url("https://test.canada.ca/canada-child-benefit-2/validation/GCWeb/images/facebook.png") no-repeat !important;
-  height: 36px;
-  width: 36px;
-  background-size: cover !important;
+.experimental .followus ul {
+  display: block;
+  list-style-type: none;
+  margin-block-start: 1em;
+  padding-inline-start: 1em;
+  font-size: 16px;
+	margin-block-end: 0em;
+}
+.experimental .followus ul li {
+  margin-bottom: 15px;
+}
+
+.experimental .followus ul li:last-child {
+  margin-bottom: 0px;
+}
+
+.experimental .followus ul li a {
+  border: none; 
+  padding: 0px 5px;
 }
-.social-lnk-2 {
+
+.experimental .social-lnk {
   position: relative; 
   bottom:-18px;
   left:45px; 
 }
-.twitter-2 {
-  background: url("https://test.canada.ca/canada-child-benefit-2/validation/GCWeb/images/twitter.png") no-repeat !important;
+
+.experimental .facebook {
+  background: url("https://test.canada.ca/canada-child-benefit-2/validation/GCWeb/images/facebook.png") no-repeat;
   height: 36px;
   width: 36px;
-  background-size: cover !important;
+  background-size: cover;
+  display: block;
 }
-.youtube-2 {
-  background: url("https://test.canada.ca/canada-child-benefit-2/validation/GCWeb/images/youtube.png") no-repeat !important;
+
+.experimental .twitter {
+  background: url("https://test.canada.ca/canada-child-benefit-2/validation/GCWeb/images/twitter.png") no-repeat;
   height: 36px;
   width: 36px;
-  background-size: cover !important;
+  background-size: cover;
+  display: block;
 }
-.instagram-2 {
-  background: url("https://test.canada.ca/canada-child-benefit-2/validation/GCWeb/images/instagram.png") no-repeat !important;
+.experimental .youtube {
+  background: url("https://test.canada.ca/canada-child-benefit-2/validation/GCWeb/images/youtube.png") no-repeat;
   height: 36px;
   width: 36px;
-  background-size: cover !important;
+  background-size: cover;
+  display: block;
 }
-
-.followus-2 {
-  line-height: 0em !important;
-}
-.followus-2 a {
-  text-decoration: none !important;
-}
-.followus-2 ul {
-  list-style-type: none !important;
-  margin-block-start: 1em;
-  padding-inline-start: 1em;
-  font-size: 16px;
-	margin-block-end: 0em;
+.experimental .instagram {
+  background: url("https://test.canada.ca/canada-child-benefit-2/validation/GCWeb/images/instagram.png") no-repeat;
+  height: 36px;
+  width: 36px;
+  background-size: cover;
+  display: block;
 }
 
-.promo-social h3 {
+.promo-social.experimental h3 {
   margin-top: 5px !important;
   text-decoration: underline !important;
 }
 
-.cust-thumbnail {
+.experimental .thumbnail {
   border-color: #335075 !important;
   border-width: 1px !important;
   padding: 0px !important;
```

Comment:
Can we avoid the use of `!important` if the class name is this specific?

Reply:
Was removed - looks like it was an out of date version you commented on?